src/applications/find-forms/components/FindVaForms.jsx

```diff
@@ -13,111 +19,147 @@ const onFeaturedContentClick = header => () => {
   });
 };
 
-export default () => (
-  <>
-    <SearchForm />
-    <SearchResults />
-    <h2>Frequently used VA forms</h2>
-    <p>
-      You can now do many form-based tasks online, like filing a disability
-      claim and applying for the GI Bill or VA health care. We&apos;ll walk you
-      through the process step-by-step.
-    </p>
-    {/* eslint-disable-next-line jsx-a11y/no-redundant-roles */}
-    <ul
-      className="usa-grid usa-grid-full vads-u-margin-top--3 vads-u-margin-bottom--4 vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row"
-      role="list"
-    >
-      <li className="featured-content-list-item vads-u-background-color--primary-alt-lightest  vads-u-padding-y--1p5 vads-u-padding-x--1p5 vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0 vads-u-display--flex vads-u-flex-direction--column">
-        <h3 className="vads-u-font-size--base vads-u-margin--0">
-          File a VA disability claim
-        </h3>
-        <hr
-          aria-hidden="true"
-          role="presentation"
-          className="featured-content-hr vads-u-margin-y--1p5 vads-u-border-color--primary"
-        />
-        <p className="va-nav-linkslist-description">
-          Equal to VA Form 21-526EZ
-        </p>
-        <a
-          className="vads-u-display--block vads-u-padding-top--1 vads-u-text-decoration--none"
-          href="/disability/file-disability-claim-form-21-526ez/"
-          onClick={onFeaturedContentClick('File a VA disability claim')}
-        >
-          <span>
-            Apply online
-            <span className="vads-u-visibility--screen-reader">
-              about filing a VA disability claim
+export const FindVaForms = ({ showPdfWarningBanner }) => {
+  useEffect(
+    () => {
+      if (showPdfWarningBanner) {
+        const PDFAlertContent = `<va-alert>
+        <h2 slot="headline" className="vads-u-font-size--h3">
+          We’re updating our forms
+        </h2>
+        <p>After January 7, 2023, you won't be able to use VA forms that have a “last updated” date before March 2022. If you downloaded any of these older VA forms, you may need to download new copies in January.</p>
```

Comment:
@wesrowe flag: the copy in our ticket doesn't match the copy Laura provided in https://github.com/department-of-veterans-affairs/va.gov-team/issues/48458#issuecomment-1320500257. Can you confirm whether our ticket copy is ok to merge, or needs to be updated to use the SWC copy provided?

Reply:
You said "okay to merge" – reminder that this **should not be merged** since that's our strategy for holding it back for Collab Cycle review.

So to your question about content...
What happened on the Search page is that SWContent proposed an update to the copy to include a "dec 15" date when all forms will have been updated, and I missed the github comment. We don't have any confirmation that is going to be the date, so I don't think we should commit to it to Vets. So for the Search page, really I think it's safe to release as it is in the PR and the issue Josh worked from. (And go to Staging Review with that version.)

Same for the Form Detail page. SWContent proposed adding the "Dec 15" – which would be a nice "next steps" detail to offer if it were safe to offer it. But we can't commit at this time to Dec 15 resolution of last few forms. So for that PR I say stick with the copy we have.

Reply:
Good catch, thx.